code/__HELPERS/globalaccess.dm

```diff
@@ -29,20 +27,6 @@
 			return global.error_cooldown;
 		if("html_interfaces")
 			return global.html_interfaces;
-		if("all_lighting_corners")
```

Comment:
What all is going on in this entire file anyway

Reply:
It's generated by a python script, see header.  I ran `python tools\gen_globals.py vgstation13.dme code\__HELPERS\globalaccess.dm` and that's what it came up with.